docs/serving/setting-up-custom-ingress-gateway.md

```diff
@@ -71,24 +58,112 @@ istio: custom-gateway
 If there is a change in service ports (compared with that of
 `istio-ingressgateway`), update the port info in the gateway accordingly.
 
-## Step 3: Update Gateway Configmap
+### Step 3: Update the gateway ConfigMap
 
-Update gateway configmap `config-istio` under `knative-serving`
+1. Update gateway configmap `config-istio` under `knative-serving`
 namespace:
 
-```bash
-kubectl edit configmap config-istio -n knative-serving
-```
-
-Replace the `istio-ingressgateway.istio-system.svc.cluster.local` field with
+     ```bash
+     kubectl edit configmap config-istio -n knative-serving
+     ```
+
+     This command opens your default text editor and allows you to edit the config-istio ConfigMap.
+
+     ```yaml
+     apiVersion: v1
+     data:
+       _example: |
+         ################################
+         #                              #
+         #    EXAMPLE CONFIGURATION     #
+         #                              #
+         ################################
+         # ...
+         gateway.knative-serving.knative-ingress-gateway: "istio-ingressgateway.istio-system.svc.cluster.local"
+     ```
+
+1. Edit the file to replace the `istio-ingressgateway.istio-system.svc.cluster.local` field with
```

Comment:
I feel "replace" might be confusing as we actually "add" a new field into "data" section.